drivers/spi/Kconfig

```diff
@@ -37,7 +37,7 @@ config SPI_EXTENDED_MODES
 
 config SPI_INIT_PRIORITY
 	int "Init priority"
-	default 70
+	default KERNEL_INIT_PRIORITY_DEVICE
```

Comment:
This small change (70 to 50) has the potential of breaking a lot of implicit dependencies on initialization priorities.

Reply:
Yeah, sorry did not mean to try sneak the change in. I'm not too attached to this change and happy to drop that patch form the stack and adjust the GPIO_SPI default accordingly if there's consensus about that - but - I can't figure a case to not set SPI devices at the same level as I2C, all drivers are either memory mapped or needs GPIO and that's it. Sure there may be some edge case like soft core or something like that, but I imagine that'd be using explicit priorities anyway, and since the change make the priority higher, I feel like it should not break much, and it'd be nice to have similar setup for bus drivers with similar purposes.

@tbursztyka fyi

Reply:
Actually, this is an old value set at a time where indeed gpio and spi could be initiated in the wrong order. Afaik, it is not the case anymore since GPIO gets initialized at KERNEL_INIT_PRIORITY_DEFAULT (which was not the case when this above value was set).

Should be fine I think